### DISCUSSION TITLE
[New Feature] Add option for spied factories to always display their cameo

### DISCUSSION THREAD

**Discussion by Rampastring** [Vinifera-Developers/Vinifera#1518] [Feature Requests]
### Description:
When you manage to sneak in a Spy to an enemy factory, you can then view what the other player is producing from that factory. To do this, you need to select the factory. However it's often tedious to keep selecting the factory during actual gameplay to check what your opponent is producing. At least for experienced players it'd be better to always display the cameo on top of the factory, regardless of selection state.

### Possible Implementation:
This could be a toggleable option in SUN.ini, `AlwaysDisplaySpiedFactoryProduction=` or something along those lines.

**Comment by CCHyper** [comment]
Maybe we could go a step further here and add a new hotkey command which toggles the view state? This way the behaviour can be toggled during the game, and without having to open the options menu to do so.
